lib/GLPI/Agent/HTTP/Client.pm

```diff
@@ -691,6 +692,13 @@ sub _KeyChain_or_KeyStore_Export {
         }
     }
 
+    # Like Mozilla::CA, but using certs from /etc/ssl/certs
+    if ($OSNAME !~ /^darwin|MSWin32$/) {
+        my $sslcacert = "/etc/ssl/certs/ca-certificates.crt";
+        push @certs, IO::Socket::SSL::Utils::PEM_file2certs($sslcacert)
+            if -e $sslcacert;
+    }
+
```

Comment:
As I said on my first comment, this is not required as this is still the default.

Reply:
I believe there should be cases where internal CA is used and some automation tool already added the self-signed certificate to system CA store but not Linux CA store. As this use case is supported on Windows and macOS (otherwise it wouldn't need to import the keystore/keychain from those systems), I don't see why don't include support for keystore of other systems too.